features/steps/ontohub_steps.rb

```diff
@@ -0,0 +1,21 @@
+When(/^I add a user to the database$/) do
```

Comment:
Please add a comment that this file is not an example for future tests and that this only tests whether or not a rollback works in general.